tox.ini

```
@@ -1,6 +1,6 @@
 [tox]
 skipsdist = True
-envlist = py37-{lint,mypy}, docs
+envlist = py37-{lint,mypy,test}, docs
```

How about running the tests in all supported python versions (#25 notwithstanding)?

```suggestion
envlist = py{34,35,36,37}-test, py37-{lint,mypy}, docs
```

Also note that this turns up a type annotation in the loader as unsupported in 3.4.

---

That's definitely something we should do but when I first turned that on I run into "mypy does not support Python 3.4 and decided to do that later and reopen #25 first. I'll take another stab at fixing the 3.4 issues.